### PR TITLE
Change url field when changing page type

### PR DESCRIPTION
**Pull request by tomas-stefano**

### CODE REVIEW

**Comment by dougdroper** [comment]
:+1:

File: app/helpers/pages_helper.rb [hunk 1]
```diff
@@ -2,4 +2,34 @@ module PagesHelper
...
+  end
+
+  private
+
+  def preview_domain
+    ComfortableMexicanSofa.config.preview_domain
+  end
+
+  def slug_tag(page_type_url, slug)
+    content_tag(:span, {data: {dough_urlformatter_url_display: true}}) do
```
Comment (benlovell) [inline app/helpers/pages_helper.rb]: You added whitespace in hash braces elsewhere but didn't here.

Comment (ourmaninamsterdam) [inline app/helpers/pages_helper.rb]: Done.

Comment (tomas-stefano) [inline app/helpers/pages_helper.rb]: @benlovell can you give a second review, after the changes, please?

**Comment by benlovell** [comment]
LGTM. :shipit:

**Comment by tomas-stefano** [comment]
Cheers mate!

**Pull request merged**